.github/workflows/db-update-qa.yml

```diff
@@ -18,31 +18,69 @@ jobs:
       db_ip: ${{ steps.get_ip.outputs.DB_IP }}
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Authenticate to Google Cloud
-        uses: google-github-actions/auth@v1
+        uses: google-github-actions/auth@v2
         with:
           credentials_json: ${{ secrets.QA_GCP_MOBILITY_FEEDS_SA_KEY }}
 
       - name: Google Cloud Setup
-        uses: google-github-actions/setup-gcloud@v1
+        uses: google-github-actions/setup-gcloud@v2
 
       - name: Get Database Instance IP
         id: get_ip
         run: |
           gcloud config set project ${{ vars.QA_MOBILITY_FEEDS_PROJECT_ID }}
-          DB_IP=$(gcloud sql instances describe ${{ secrets.DB_INSTANCE_NAME }} --format=json | jq -r '.ipAddresses[] | select(.type=="PRIMARY") | .ipAddress')
+          DB_IP=$(gcloud sql instances describe ${{ secrets.DB_INSTANCE_NAME }} --format=json | jq -r '.ipAddresses[] | select(.type=="PRIVATE") | .ipAddress')
           echo "DB_IP=$DB_IP" >> "$GITHUB_OUTPUT"
+  gw_status:
```

Comment:
suggestion: extract the `gw_status` as a separate callable workflow

Reply:
Let's do this in the future to unblock all other work waiting for this change